### ISSUE TITLE
Feature Request For Navigation Icons

### ISSUE THREAD

**Issue by CrypticCus** [easymodo/qimgv#382]
Can we have an option to enable navigation icons in options? thanks.

**Comment by easymodo** [comment]
which icons do you mean? for filesystem navigation like back/forward?

**Comment by CrypticCus** [comment]
yes because when i use scrool wheel for zoom in and out, cant use mouse to navigate. also if u added overscrooling feature may i have test zip like u sended for ctrl+v option

**Comment by CrypticCus** [comment]
I mean this button at both side, when you hover mouse on them... also they are not visible right now because i enabled LOOP in the settings. to enable them for mouse hovering would be cool

![image](https://user-images.githubusercontent.com/93516630/147568942-700ef078-6d23-4902-a54b-06a774bce651.png)

**Comment by easymodo** [comment]
yeah i got it
i'll probably just make the edges clickable instead of adding buttons

**Comment by CrypticCus** [comment]
even better. btw i'm gonna make one more simple suggestion since i think i understand your GUI taste. this is windows media player classic of mine. it doesnt have title part

![image](https://user-images.githubusercontent.com/93516630/147582200-56e66f2a-3484-476c-b478-5f33f8984844.png)

**Comment by CrypticCus** [comment]
BTW can we also have an option so that videos have sound by deault or starts muted? right now they are muted by default

and i can't find where does settings are saved? i want to export and import my settings.

**Comment by easymodo** [comment]
there is already a checkbox for sound playback
settings are in conf folder (on windows)
